pwndbg/heap/structs.py

```diff
@@ -594,6 +657,63 @@ class c_malloc_par_2_23(Structure):
         ("sbrk_base", c_pvoid),
     ]
 
+class c_malloc_par_2_12(Structure):
+    """
+    This class represents the malloc_par struct for GLIBC < 2.15 as a ctypes struct.
```

Comment:
Just to be sure: how do we know 2.15 is correct here? If we really know it, can we link to diff or code and state what has changed?

Reply:
Indeed that "no description provided" for this PR is killing me